microsoft-365/admin/email/create-a-shared-mailbox.md

```diff
@@ -88,6 +88,9 @@ You can use the following permissions with a shared mailbox:
 
 - **Send on Behalf**: The Send on Behalf permission lets a user send email on behalf of the shared mailbox. For example, if John logs into the shared mailbox Reception Building 32 and sends an email, it will look like the mail was sent by "John on behalf of Reception Building 32". You can't use the EAC to grant Send on Behalf permissions, you must use the **Set-Mailbox** cmdlet with the _GrantSendonBehalf_ parameter.
 
+> [!NOTE]
+> **Send As** and **Send on Behalf** do not work in Outlook Desktop client with **HiddenFromAddressListsEnabled** parameter on the mailbox set to **True**, since they require  the mailbox to be visible in Outlook via Global Address List.
```

Comment:
```suggestion
> The **Send As** and **Send on Behalf** permissions do not work in Outlook Desktop client with the *HiddenFromAddressListsEnabled* parameter on the mailbox set to **True**, since they require the mailbox to be visible in Outlook via the Global Address List.
```